include/itkPhaseCorrelationOptimizer.hxx

```diff
@@ -225,7 +225,7 @@ PhaseCorrelationOptimizer<TRealPixelType, VImageDimension>::ComputeOffset()
   const ImageType *        fixed = static_cast<ImageType *>(this->GetInput(0));
   const ImageType *        moving = static_cast<ImageType *>(this->GetInput(1));
   const ImageType *        input = static_cast<ImageType *>(this->GetInput(2));
-  const ComplexImageType * complexInput = static_cast<ComplexImageType *>(this->GetInput(3));
+  //const ComplexImageType * complexInput = static_cast<ComplexImageType *>(this->GetInput(3));
```

Comment:
I preferred to comment this line instead of removing it since I ignore whether it was left as an indicative of ToDo work on the filter.

Reply:
At least add a space after the comment, so linter passes. But it is probably better to remove this, as it is a leftover from a refactoring.

Reply:
:+1: Removed in 70c206a . Thanks for the review @dzenanz .